app/src/main/java/org/mozilla/fenix/perf/ProfilerReusableComposable.kt

```diff
@@ -40,42 +44,46 @@ fun ProfilerDialogueCard(content: @Composable () -> Unit) {
 }
 
 /**
- * Top level radio button for the profiler dialogue
+ * Top level radio button for the profiler dialogue.
+ *
+ * @param text The main text to be displayed.
+ * @param subText The subtext to be displayed.
+ * @param selected [Boolean] that indicates whether the radio button is currently selected.
+ * @param onClick Invoked when the radio button is clicked.
  */
 @Composable
 fun ProfilerLabeledRadioButton(
     text: String,
     subText: String,
-    state: MutableState<String>,
+    selected: Boolean = false,
+    onClick: () -> Unit,
 ) {
-    Row {
+    Row (
+        modifier = Modifier.clickable { onClick() },
+    ) {
         RadioButton(
-            selected = state.value == text,
-            onClick = { state.value = text },
+            selected = selected,
+            onClick = {},
             enabled = true,
         )
         Column {
```

Comment:
Could we add the padding in column to avoid repeating it in the 2 Text below?

Reply:
I would also imagine we should also do `padding(horizontal = 8.dp)` to help balance things when it comes to RTL support.

Reply:
fixed